src/test/java/com/victuxbb/systemdesigns/tinyurlkgs/infrastructure/api/KGSRestControllerTest.java

```diff
@@ -0,0 +1,23 @@
+package com.victuxbb.systemdesigns.tinyurlkgs.infrastructure.api;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class KGSRestControllerTest {
```

Comment:
Remove that class we will do it later, when we close the contracto for the API

Reply:
yes, the contracto is importante!